test/test_parse_psse.jl

```diff
@@ -36,4 +36,15 @@ end
     @test get_available(get_component(ThermalStandard, sys, "generator-2438-ND")) == 0
     @test get_status(get_component(ThermalStandard, sys, "generator-2438-EG")) == 1
     @test get_available(get_component(ThermalStandard, sys, "generator-2438-EG")) == 1
+
+    @info "Testing Load Zone Formatter"
+    PSB.clear_serialized_systems("psse_Benchmark_4ger_33_2015_sys")
+    sys3 = build_system(
+        PSYTestSystems,
+        "psse_Benchmark_4ger_33_2015_sys";
+        loadzone_name_formatter = x -> string(3 * x),
+    )
+    lz_original = first(get_components(LoadZone, sys2))
```

Comment:
I don't think you can safely assume that the result of `get_components` will be in the same order every time. Maybe get all the `LoadZone` names in each, sort those lists, and compare the lists?

Reply:
There is only one load zone for that system, that's why I can use first

Reply:
Oh okay